test_runner/fixtures/benchmark_fixture.py

```diff
@@ -170,7 +170,7 @@ def record(
                 "name": metric_name,
                 "value": metric_value,
                 "unit": unit,
-                "report": report,
+                "report": str(report),
```

Comment:
This is needed for `pytest-xdist` to run tests in parellel. Otherwise, run `pytest ... -n4` will return `execnet.gateway_base.DumpError: can't serialize <enum MetricReport>` error.